server/controllers/notification.js

```diff
@@ -0,0 +1,63 @@
+const asyncHandler = require("express-async-handler");
+const Notification = require("../models/Notification");
+const User = require("../models/User");
+
+exports.createNotification = asyncHandler(async (req, res, next) => {
+  const body = req.body;
+  try {
+    const user = await User.findById(req.user.id);
+    const notification = await Notification.create({
+      to: body.to,
+      from: req.user.id,
+      notification: body.notification,
+      profilePic: user.profilePic,
```

Comment:
Aren't we storing the user's profile pic in the Profile model?

Reply:
key value pair removed